scripts/prep-t2m-v4.py

```
@@ -0,0 +1,307 @@
+            labels[mid] = forced; continue
+        hits = {KEYWORDS[k] for k in KEYWORDS if re.search(r"\b" + k, desc)}
+        if len(hits) == 1:
+            labels[mid] = hits.pop()
+    return labels
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--bvh", required=True)
+    ap.add_argument("--index", required=True)
+    ap.add_argument("--out", default="/tmp/t2m_v4.npz")
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/t2m_v4.npz"

(S108)